src/components/calculator.js

```diff
@@ -1,119 +1,106 @@
 /* eslint-disable react/prefer-stateless-function */
 
-import React, { Component } from 'react';
+// import React, { Component } from "react";
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-export default class calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: '0',
-      operation: null,
-      next: null,
-    };
-  }
+// export default class calculator extends Component {
+//   constructor(props) {
+//     super(props);
+//     this.state = {
+//       total: '0',
+//       operation: null,
+//       next: null,
+//     };
+//   }
```

Comment:
- [ ] Kindy remove commented-out code from your code to keep your code base clean